config/default.yml

```diff
@@ -199,6 +199,16 @@ Rails/ContentTag:
   Exclude:
     - app/models/**/*.rb
 
+Rails/ControllerTesting:
```

Comment:
As you mention it in the cops' `@safety` doc, it makes sense to mark it as `SafeAutocorrect: false`.